financial-connections/res/values/strings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="consent_pane_tc">You agree to Stripe\'s <annotation clickable="terms">Terms</annotation> and <annotation clickable="privacy">Privacy Policy</annotation>. <annotation clickable="more">Learn more</annotation></string>
+    <string name="stripe_consent_pane_title">%1$s works with Stripe to link your accounts</string>
+    <string name="stripe_consent_pane_body1">Only the <annotation clickable="data">data requested</annotation> is shared with %1$s. We never share your login details with them.</string>
+    <string name="stripe_consent_pane_body2">Your data is encrypted for your protection.</string>
+    <string name="stripe_consent_pane_body3">You can <annotation clickable="disconnect">disconnect</annotation> your accounts at anytime.</string>
+    <string name="stripe_consent_pane_agree">Agree</string>
```

Comment:
Are these going to be translated via lokalise?

Reply:
For now we just support en-US, but preparing this for a future where localization comes in : )

Reply:
Okay, we had issues with lokalise in the past to support both android and ios strings that contain links. What I have seen is two translated strings with android annotations and ios annotations.